ocaml/default.nix

```diff
@@ -232,6 +232,20 @@ in
       };
     });
 
+    luv = buildDunePackage {
```

Comment:
I'd prefer to have these definitions in their own folder. otherwise this file just becomes too cluttered.